lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -103,7 +103,9 @@ void DAP::SendJSON(const llvm::json::Value &json) {
   SendJSON(json_str);
 
   if (log) {
-    *log << "<-- " << std::endl
+    auto now = std::chrono::duration<double>(
+        std::chrono::system_clock::now().time_since_epoch());
```

Comment:
wouldn't it be better to use steady_clock? that's more stable for benchmarking

Reply:
for a benchmark yes, but this is only a "benchmark" (with *very* large quotes). Like, I won't be using this during daylight savings switchover, and I don't think it's suitable for anything automated.  And it's kinda nice it matches the time stamps for regular lldb logs (I literally copied this code from there).